estela-web/src/pages/ProjectDashboardPage/index.tsx

```diff
@@ -24,7 +31,7 @@ interface ProjectDashboardPageState {
     count: number;
     current: number;
     loadedStats: boolean;
-    globalStats: GlobalStats[];
+    projectStats: ProjectStats[];
```

Comment:
I have found more instances of `GlobalStats` in `src/components/Stats/ProjectHealth.tsx` and `src/components/Stats/StatsTableSection.tsx` in case those should also be replaced.

Reply:
I deleted that instances.